webview-ui/src/components/settings/TemperatureControl.tsx

```diff
@@ -48,27 +49,15 @@ export const TemperatureControl = ({ value, onChange, maxValue = 1 }: Temperatur
 					}}>
 					<div style={{ display: "flex", alignItems: "center", gap: "5px" }}>
 						<input
-							aria-label="Temperature control text input"
-							type="text"
+							type="range"
```

Comment:
Actually I'm not sure about changing this to a slider - not all of the providers have the same 0-2 range. And separately, I'm noticing an error where the value gets reset to 1.

https://github.com/user-attachments/assets/be3fcdcf-0f18-4a66-8488-e4492f76e13e

Reply:
Can we just leave as a text input for now?

Reply:
sure. 1 is the default value, other values ​​can also be the default values

Reply:
range 0-2 comes from the parent component, and default value is set when isCustomTemperature is toggled, these behaviors are the same as before the change, is it really bad after changing to a slider?

Reply:
https://github.com/user-attachments/assets/ef727be2-2038-42b4-842e-20dbdc984c01

Reply:
Ah, I guess I was confused since 0 used to be the default value. I don't feel super strongly against the slider and it's certainly nicer in some ways, I just think it implies a valid range of 0-2 that's not the case for all models.

Reply:
> Ah, I guess I was confused since 0 used to be the default value. I don't feel super strongly against the slider and it's certainly nicer in some ways, I just think it implies a valid range of 0-2 that's not the case for all models.

I'm not sure either, I didn't change maxValue, and I've restored the default value to 0

Reply:
If you like the slider better let’s go for it then